### ISSUE TITLE
Strange artifacts present with Quest firmware V25+

### ISSUE THREAD

**Issue by ghost** [Powerbyte7/seurat_capture_addon#6]
When I load the environment on the Quest I get these spikey artifacts that don't show in Blender.

I'm not sure if this is being caused because of the current Quest version or not. It seemed fine in the video you posted on the Quest Homes Discord.

![com oculus shellenv-20210302-083456](https://user-images.githubusercontent.com/66656514/109663684-605d9f00-7b3a-11eb-85b1-b1247c12d351.jpg)

![com oculus shellenv-20210302-083503](https://user-images.githubusercontent.com/66656514/109663690-618ecc00-7b3a-11eb-8b3b-97e6e9a6e83d.jpg)

**Comment by Powerbyte7** [comment]
Yeah, I have these artifacts too, it seems to be present in V25 and later. We'll need to wait until Oculus pushes a fix, I'll pin this issue in the meantime.

**Comment by ghost** [comment]
I seems like switching on Alpha Blend in Blender fixes those black triangles. They can still be seen at some angles, and the hatching effect is still there.

**Comment by Powerbyte7** [comment]
@JaidynHolcomb Okay, it looks like V27 has fixed some of the major issues with environments, like performance and alpha clipping. The 'hatching' effect is still present unfortunately, but as it turns out you can fix it with a quick change to the UVs, this fix works for V25 and above:

1. Select your mesh and go into the UV editor
2. Select all faces
3. Set the scaling mode to 'Individual Origins'
4. Scale the faces down by 0.9

![image](https://user-images.githubusercontent.com/40431794/112862955-9c8f0b80-90b6-11eb-8ac4-0ec9a0e934bb.png)

I'll keep this issue pinned for now, in case anyone else is looking  for a fix.
